packages/eslint-plugin/src/configs/base.js

```diff
@@ -19,10 +19,9 @@ module.exports = {
   },
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the FluentUIV0 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33448&commitId=826bb409967f86a001dbedeb43edc329945190ce&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.CardExampleSimple.chromium.png|224| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>